### ISSUE TITLE
djangoUrl relative vs absolute url

### ISSUE THREAD

**Issue by sssolid** [mesemus/django-angular-dynamic-forms#3]
The '**djangoUrl**' returned from the server is relative and not absolute.  This has caused an issue getting form submissions to work correctly in a development environment as it would attempt to contact the frontend instead of the backend.

I have changed the following to return an absolute url.  Not entirely sure if this is an issue or I have overlooked a simple fix.

**Line 326 in 'rest.py'**

Original
```ret['djangoUrl'] = base_path + self._get_url_by_form_id(form_name)```

Changed
```ret['djangoUrl']` = self.request.build_absolute_uri(base_path + self._get_url_by_form_id(form_name))```

**Comment by mesemus** [comment]
@sssolid : thanks for the issue. In my development scenario I do not have it as I'm using angular cli with proxying on, that is:

`ng serve --proxy-config proxy.conf.json`
... running on 127.0.0.1:4200

```
cat proxy.conf.json 
{
    "/api": {
        "target": "http://127.0.0.1:8010",
        "secure": false,
        "logLevel": "debug"
    }
}
```
which for example proxies `http://127.0.0.1:4200/api/blah` to `http://127.0.0.1:8010/api/blah`. 

In production the setup is similar with nginx handling angular files and proxying django.

During the weekend I'll look if this change is ok with our deployment scenario 
(I'm not sure it the development proxy passes the port&path correctly) and will
add absolute/relative urls as an option to django settings.

Thanks again, ms.

**Comment by mesemus** [comment]
Fixed in 6.1.2, I will have to write some tests for that :)
